.github/workflows/nightly.yml

```diff
@@ -48,7 +48,16 @@ jobs:
           echo ::set-output name=NEXT_VERSION::${NEXT_VERSION}
 
           git checkout -b nightly/${NEXT_VERSION}
-          npm version ${NEXT_VERSION}
+          
+          if [ -z "$(git status --porcelain)" ]; then
+            echo "✨"
+          else
+            echo "dirty working directory..."
+            git add .
+            git commit -m "dirty working directory..."
```

Comment:
Why is there a dirty working directory? What steps trigger file changes?

Reply:
great question, that i'd love to have an answer for!

in my test repo i was getting a changed package-lock.json, but not sure what the dirtiness is here.

might be fine to leave the commit for a day and see if we can remove it tomorrow after we know what changed?

Reply:
Sounds good 👍